app/assets/javascripts/app/models/post/interactions.js

```diff
@@ -1,19 +1,15 @@
 // @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-v3-or-Later
 
-//require ../post
+//= require ./like_interactions
 
-app.models.Post.Interactions = Backbone.Model.extend({
+app.models.Post.Interactions = app.models.Post.LikeInteractions.extend({
   initialize : function(options){
+    app.models.Post.LikeInteractions.prototype.initialize({post : this.post}); // I don't know how to call the super method yet
```

Comment:
- Line 7 exceeds the maximum line length of 120.
- Extra space after key 'post'.